spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+describe "POST api/v1/targets", type: :request do
+
+  let(:topic_list) { create_list(:topic, 5) } 
+  let(:user) { create(:user) }
+
+  let(:params) do {
+    user: {
+      email: user.email,
+      password: user.password
+    }    
+  }
+  end
+
+  before do
+    user.confirm
```

Comment:
Instead of doing this, try setting the `confirmed_at` attribute at the user factory.